x/ecocredit/server/basket/create_test.go

```diff
@@ -166,9 +147,7 @@ func TestValidBasket(t *testing.T) {
 		Metadata:         "",
 		CreditTypeAbbrev: "C",
 	}))
-	s.paramsKeeper.EXPECT().GetParamSet(gmAny, gmAny).Do(func(any interface{}, p *core.Params) {
-		p.BasketFee = fee
-	}).Times(1)
+	utils.ExpectParamGet(&basketFees, s.paramsKeeper, core.KeyBasketCreationFee, 1)
```

Comment:
Here's a working example without using `gomock.Any()` and using a specific time:

```suggestion
	var fee sdk.Coins
	s.paramsKeeper.EXPECT().Get(s.sdkCtx, core.KeyBasketCreationFee, &fee).
		Do(func(ctx sdk.Context, key []byte, ptr *sdk.Coins) {
			ptr = &basketFees
		}).
		Times(1)
```

Reply:
Not sure if setting the ptr is necessary because it works without it, in which case it could be:

```suggestion
	var fee sdk.Coins
	s.paramsKeeper.EXPECT().Get(s.sdkCtx, core.KeyBasketCreationFee, &fee).
		Do(func(sdk.Context, []byte, *sdk.Coins) {}).
		Times(1)
```